DoSomething/intertwine
# Editor data added to export + used for imports

You'll want to wait on #25 to get merged in first before reviewing this one. The diff will be a lot easier to navigate then.
#### What's this PR do?

Our SMS game json configs in their current state don't have enough information to fully recreate the Intertwine workspace the story was created in. This PR adds that data (text, passage position) to the export.
#### Where should the reviewer start?
###### exportsms.js
- For the story config passage, position data is added to `_twinedata.storyconfig.pos.top/.left`
- For the DS passage, position data is added to each story node at `_twinedata.pos.top/.left`
- The full text of the passage is also added for DS passage at `_twinedata.text`
###### importsms.js
- Story config and DS passages pull this new exported data and use it for creating the new models.
#### How should this be manually tested?
- Create a story and "Export for SMS"
- Save the export as a something.json file
- Import that something.json file
- Verify the imported story's content and workspace match the exported content and workspace
#### What are the relevant tickets?

Closes #26
